doc/api/domain.md

```diff
@@ -436,7 +436,7 @@ without exiting the domain.
 
 ### domain.dispose()
 
-    Stability: 0 - Deprecated.  Please recover from failed IO actions
+> Stability: 0 - Deprecated.  Please recover from failed IO actions
```

Comment:
The next line also should be converted to `>` or at least the padding should be removed. Else, this is how it renders:
![screenshot_20160720_200119](https://cloud.githubusercontent.com/assets/291301/16995405/e57efdf8-4eb4-11e6-85e0-b39237f6e637.png)